stackit/internal/validate/validate.go

```diff
@@ -225,3 +226,30 @@ func CIDR() *Validator {
 		},
 	}
 }
+
+func Rrule() *Validator {
```

Comment:
Could you also add unit tests to this validator? You can find examples for all others in `validate_test.go`